client/router.coffee

```diff
@@ -48,7 +44,7 @@ Router.route "/admins",
   onBeforeAction: ->
     redirectIfNotAuthorized(@, ['admin'])
   waitOn: ->
-    Meteor.subscribe "allUsers"
+    [Meteor.subscribe("allUsers"), Meteor.subscribe("roles")]
```

Comment:
The redirectIfNotAuthorized function in the router depends on this subscription so moving it might require more refactoring than you would expect.